kubernetes/provider.go

```diff
@@ -213,6 +213,8 @@ func Provider() *schema.Provider {
 			"kubernetes_all_namespaces":             dataSourceKubernetesAllNamespaces(),
 			"kubernetes_secret":                     dataSourceKubernetesSecret(),
 			"kubernetes_secret_v1":                  dataSourceKubernetesSecret(),
+			"kubernetes_endpoints":                  dataSourceKubernetesEndpoint(),
```

Comment:
For new resources and data sources, we use names with the API versions only. Please, remove `kubernetes_endpoints` and leave only `kubernetes_endpoints_v1`.